lib/recite.rb

```diff
@@ -1,6 +1,32 @@
 # frozen_string_literal: true
 class Recite
   def recite
-    File.read("lib/assets/nursery_rhyme.txt")
+    lines = File.read("lib/assets/nursery_lines.txt").split("\n")
+    build_rhyme(lines)
+  end
+
+  def random_recite
+    # shuffled array of phrases in the nursery rhyme
+    shuffled = File.read("lib/assets/nursery_lines.txt").split("\n").shuffle
+
+    # construct random nursery rhyme
+    build_rhyme(shuffled)
+  end
+
+  # helper to create a nursery rhyme from independent lines
+  def build_rhyme(lines)
+    rhyme = ""
+    lines.length.times do |index|
```

Comment:
issue: The standard way to get an array with an index is to use a method like `.each_index`